libmscore/album.cpp

```diff
@@ -193,14 +271,68 @@ bool Album::scoreInActiveAlbum(MasterScore *score)
 
 MasterScore *Album::getDominant()
```

Comment:
Also, this can probably be a `const` method. I completely forgot about this last time I reviewed, but I think their are probably a few other methods that need to change in the same way. I'm sure you can figure out what they are though, so I'll stop commenting it.

In general, a method should be `const` if it functions as a 'getter', i.e. if it returns a value calculated from the class state. If it has a side effect (e.g. changing a flag, fixing some internal structure), it doesn't need to be const, but consider renaming it to make its purpose clearer. I don't know how applicable that is here,